### PR TITLE
fix requires

### PR DESCRIPTION
**Pull request by cmal**
I found `re-frame` use some clojure official libraries with the unstrict format, that make it impossible to be used with some tools like `shadow-cljs` and I fixed these.

Sorry for the space/format change if you may mind that, my Emacs automatically did that 😄  .

### CODE REVIEW

File: src/re_frame/interceptor.cljc [hunk 1]
```diff
@@ -1,7 +1,9 @@
 (ns re-frame.interceptor
   (:require
-    [re-frame.loggers :refer [console]]
-    [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]))
+   [re-frame.loggers :refer [console]]
+   [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]
+   [clojure.set :as set]
+   ))
```
Comment (thenonameguy) [inline src/re_frame/interceptor.cljc]: I think the closing parens should be at the end of the above line.

Comment (cmal) [inline src/re_frame/interceptor.cljc]: Ahh. sorry for that.

Comment (cmal) [inline src/re_frame/interceptor.cljc]: Excuse me, I am not quite sure about what you mean. Could you tell me the right way so that I can fix it. Thanks.

Comment (thenonameguy) [inline src/re_frame/interceptor.cljc]: It should be:
```clojure
[clojure.set :as set]))
```
instead of:
```clojure
[clojure.set :as set]
))
```

Comment (ruiyang) [inline src/re_frame/interceptor.cljc]: I think he means
it should be like the following at the end of the require.
 [clojure.set :as set]))

Comment (cmal) [inline src/re_frame/interceptor.cljc]: Sorry for that, I'll change this.

**Pull request closed without merging**